src/System.IO.FileSystem/src/System/IO/FileSystem.Windows.cs

```diff
@@ -328,7 +328,7 @@ public static DateTimeOffset GetLastWriteTime(string fullPath)
 
         public static void MoveDirectory(string sourceFullPath, string destFullPath)
         {
-            if (!Interop.Kernel32.MoveFile(sourceFullPath, destFullPath))
+            if (!Interop.Kernel32.MoveFile(sourceFullPath, destFullPath, false))
```

Comment:
Nit: please name the `false` argument so it's clear at the call site what the `false` means.

Reply:
Change made.